auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -304,8 +144,8 @@ public static AuthUI getInstance(FirebaseApp app) {
     }
 
     /**
-     * Default theme used by {@link SignInIntentBuilder#setTheme(int)} if no theme
-     * customization is required.
+     * Default theme used by {@link SignInIntentBuilder#setTheme(int)} if no theme customization is
```

Comment:
Just a note for the future, including formatting changes like this that are unrelated to the functionality of the PR is distracting.  I can normally look around it, but just wanted to point it out.

Is it your IDE settings that do this automatically?